i18n/fr.yaml

```
@@ -106,6 +107,9 @@ collections:
     title: Mémos
     subtitle: Un mémo est une courte analyse de changements significatifs qui explique clairement leurs conséquences pour les utilisateurs.
     cta: Voir tous les mémos
+  reuses:
+    title: Réutilisations
+    subtitle: Des réutilisations identifiées des données de cette collection.
```

```suggestion
    subtitle: Réutilisations connues des données de cette collection.
```

Ou `recensées` à la place de `connues`?